src/track/globaltrackcache.h

```diff
@@ -229,15 +229,17 @@ class GlobalTrackCache {
             const TrackRef& trackRef,
             TrackId trackId);
 
-    Track* evictInternal(
-            const TrackRef& trackRef,
-            Track* pTrack,
-            bool evictUnexpired);
-
     void evict(
             Track* pTrack,
             bool evictUnexpired = false);
 
+    typedef std::set<Track*> AllocatedTracks;
+
+    bool evictInternal(
```

Comment:
Should this be private?

Reply:
It is already _private_ like most functions.

Reply:
Oh, sorry for the noise. I got confused by the `public` keyword for the Item class.